bin/create_folders.py

```diff
@@ -32,9 +34,57 @@ def create_folder(
     return subfolder
 
 
-# TODO: https://sagebionetworks.jira.com/browse/IBCDPE-809
-# def update_subfolders():
-#     """"""
+def prefix_filename(
+    syn: synapseclient.Synapse, prefix_name: str, old_file_entity: synapseclient.File
+) -> None:
+    """
+    Prefixes the name of the old file entity with the desired ``prefix_name`` and updates the file name and metadata in Synapse.
+
+    Arguments:
+        syn: The Synapse Python client instance
+        submission_id: The ID of the submission
+        old_file_entity: The old file entity to be updated
+
+    """
+    filename = old_file_entity.name
+    predictions_file_name = f"{prefix_name}_{filename}"
+
+    updated_file_entity = synapseutils.changeFileMetaData(
+        syn,
+        entity=old_file_entity,
+        downloadAs=predictions_file_name,
+        forceVersion=False,
+    )
+    updated_file_entity.name = predictions_file_name
+    syn.store(updated_file_entity)
+
+
+def update_subfolders(
+    syn: synapseclient.Synapse,
+    predictions_file: str,
+    submitter_id: str,
+    parent_id: Union[str, synapseclient.Entity],
+) -> synapseclient.File:
+    """
+    Update subfolders based on the given predictions file, submitter ID, and parent ID, and returns the file entity.
+
+    Arguments:
+        syn: A Synapse Python client instance
+        predictions_file: The name of the predictions file
+        submitter_id: The ID of the submitter
+        parent_id: The ID of the parent folder
+    """
+    submitter_folder = syn.findEntityId(submitter_id, parent_id)
+
+    for folder in syn.getChildren(submitter_folder, includeTypes=["folder"]):
```

Comment:
You can make a change here to work differently:
```
submitter_folder = syn.findEntityId(submitter_id, parent_id)

predictions_folder = syn.findEntityId("predictions", submitter_folder )
```

Reply:
Good catch, thanks!